wsgi_kerberos.py

```diff
@@ -51,28 +58,29 @@ def _consume_request(environ, read_max):
         return
     if environ.get("HTTP_EXPECT") == "100-continue":
         return
+    if environ["REQUEST_METHOD"] in _NEVER_READ_METHODS:
+        return
     try:
-        sock = environ.get('wsgi.input')
```

Comment:
(`wsgi.input` is never a socket object, it is always a file-like object. For example, it has a `read()` method rather than a [`recv()`](https://docs.python.org/3/library/socket.html#socket.socket.recv) method. Reduce confusion by renaming this `sock` variable.)